res/values/strings.xml

```diff
@@ -36,4 +36,16 @@
     <string name="swipe_gestures_title">Swipe to navigate your phone</string>
     <string name="swipe_gestures_desc">Learn gestures to go home, go back and switch between apps</string>
     <string name="try_it">Try it</string>
+    <string name="oem_unlocked_warning">The boot loader is unlocked and software integrity can not be guaranteed. Any data stored on this device may be available to attackers. Do not store any sensitive data on this device.</string>
```

Comment:
the underlying api uses this terminology: https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/service/oemlock/OemLockManager.java#155

the api for determining bootloader unlocked status is called `OemLockManager.isDeviceOemUnlocked()`.

so it can be looked at this way:
- oem unlocked
- oem unlocking enabled

the above 2 are different things (as per upstream terminology)